packages/react-select/src/components/Menu.tsx

```diff
@@ -594,23 +599,12 @@ export const MenuPortal = <
   const menuPortalRef = useRef<HTMLDivElement | null>(null);
   const cleanupRef = useRef<(() => void) | void | null>(null);
 
-  const [placement, setPlacement] = useState<'bottom' | 'top'>(
+  const [placement, setPortalPlacement] = useState<'bottom' | 'top'>(
     coercePlacement(menuPlacement)
   );
   const [computedPosition, setComputedPosition] =
     useState<ComputedPosition | null>(null);
 
-  // callback for occasions where the menu must "flip"
-  const getPortalPlacement = useCallback(
-    ({ placement: updatedPlacement }: CalculatedMenuPlacementAndHeight) => {
-      // avoid re-renders if the placement has not changed
-      if (updatedPlacement !== placement) {
```

Comment:
I don't think we need this check because React knows to bail out if the value did not change.